core/modules/filters.js

```diff
@@ -82,7 +82,7 @@ function parseFilterOperation(operators,filterString,p) {
 				if(rexMatch) {
 					operator.regexp = new RegExp(rexMatch[1], rexMatch[2]);
 // DEPRECATION WARNING
-console.log("WARNING: Filter",operator.operator,"has a deprecated regexp operand",operator.regexp);
+console.log("\nWARNING: Filter",operator.operator,"has a deprecated regexp operand",operator.regexp);
```

Comment:
Why is this newline being introduced?

Reply:
If I run the tests like so: 

```
PS D:\git\tiddly\tiddlywiki\TiddlyWiki5> node .\tiddlywiki.js .\editions\test\ --build index
```
I get this output, which is much better to read, then with no NL and it doesn't create problems in the browser console. The warning is more visible there too. 

```
Randomized with seed 01155
Started
.....................................................................
WARNING: Filter modifier has a deprecated regexp operand /JoeBloggs/

WARNING: Filter modifier has a deprecated regexp operand /Jo/
......................................................
WARNING: Filter modifier has a deprecated regexp operand /JoeBloggs/

WARNING: Filter modifier has a deprecated regexp operand /Jo/
..........................


149 specs, 0 failures
Finished in 0.822 seconds
Randomized with seed 01155 (jasmine --random=true --seed=01155)
```

Without the new-lines the test-output is "scary". IMO with the NLs it's just a warning, which can be ignored.

Reply:
With no new-lines, the first thing I saw, was the red text `/JoeBloggs/` and `/Jo/`. The rest of the text is hard to read. So I thought there is a real problem going on. ... 

The NL may avoid this problem for other users in the future.

Reply:
Hi @pmario that makes sense, but it's still troubling to be changing something unrelated to the core purpose of this PR. I'd prefer to keep it how it is, or to add an option for disabling the warning message.